salt/modules/acme.py

```diff
@@ -360,4 +371,13 @@ def needs_renewal(name, window=None):
     if window is not None and window in ('force', 'Force', True):
```

Comment:
It's not a problem with your code, but if you'd like to improve this, this seems silly. It's already doing a membership check, and `None` is not one of those options. Though looking at it I wonder if at one point it was `if window is not None and window.lower() in (...)`. It seems to me like this would be better as `if str(window).lower() in ('force', 'true'):`, because that's a lot more simple.

Reply:
Indeed. Included this change in 8fce3b1d3a100ebb5afa3c97c7ced2c2735437d3.